web/src/ui_init.js

```diff
@@ -663,6 +664,7 @@ export function initialize_everything(state_data) {
 
     i18n.initialize({language_list: page_params.language_list});
     timerender.initialize();
+    information_density.initialize();
```

Comment:
Should we immediately after `initialize_user_settings`, maybe before the `dark_theme.enable` block for spectators? While the i18n and timerender initialization functions don't do much, I think that might better clarify that this is a base detail we want to do before essentially all other initialization.

Reply:
That's a good point. I'll shift this up.